lib/App/DuckPAN/Cmd/Test.pm

```diff
@@ -1,6 +1,7 @@
 package App::DuckPAN::Cmd::Test;
 # ABSTRACT: Command for running the tests of this library
 
+no warnings 'uninitialized';
```

Comment:
Thanks 👍 

I haven't been able to produce those warnings -- how did you do it?